src/rez/pip.py

```diff
@@ -137,7 +137,7 @@ def find_python_in_context(context):
     # accidentally find a system python install.
     #
     context = context.copy()
-    context.append_sys_path = False  # https://github.com/nerdvegas/rez/issues/826
+    context.append_sys_path = False  # #826
```

Comment:
Taken out of context, it would be hard to know that this is a GitHub issue number.

Reply:
Agreed. It'd be nice if github had some smart syntax for issues that it could generate links on in its md previews :/. Perhaps we just write these like `issue-826` / `pr-826`, it'd be nice to standardize so they're greppable. I want to keep away from overly verbose full links.

Reply:
I like full links as I can just copy/paste and it gets me straight to where I want. If we go with shorthands, it will require more clicks and steps for little benefits, considering there is only 12 links to issues in this PR.

But I also wish allowed us to link to other links in the repo from any Markdown files without relying on full URLs...

Reply:
But I can live with `issue-####` if you prefer that.

Reply:
I meant what you mentioned there ^ ie some sort of relative link for the issue/pr, that'd be a lot cleaner than a full link.

Reply:
We can always buy a domain and make it redirect to GH...